cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1328,6 +1329,11 @@ def setUp(self):
         super(TestComponentTemplates, self).setUp()
         self.templates = get_component_templates(self.course)
 
+        # Initialize the deprecated modules settings with empty list
+        XBlockDeprecatedAdvancedComponentConfig.objects.create(
```

Comment:
This is not needed. `XBlockDeprecatedAdvancedComponentConfig` should be returning an empty list by default if no model exists.